charts/argo-workflows/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.6.0
+version: 0.6.1
```

As this adds a feature, please bump the minor version.
```suggestion
version: 0.7.0
```

---

Resolved, as per PR update